client/src/components/MyBusinessList/MyBusinessList.tsx

```diff
@@ -11,7 +11,7 @@ import '../../pages/Profile/Profile.css'
 export const MyBusinessList = () => {
   const [list, setList] = useState<any[]>([])
   const [loading, setLoading] = useState(true)
-  const user = JSON.parse(localStorage.getItem('profile') || '{}').data.result; 
+  const user = JSON.parse(localStorage.getItem('profile') || '{}').result; 
```

Comment:
Hi @sergiobtos!  Thank you for your review.  In response to your [comment](https://github.com/Brridge-Group/Lions-Gate-Hairr/pull/34#pullrequestreview-980800667).  I have modified this line of code due to the associated terminal error: `Uncaught TypeError: can't access property "result," JSON.parse(...).data is undefined`.  See enclosed image.  After debugging, I located the data elsewhere.  The modification restored the blank profile page to the intended profile page UI.  I also conferred with @cynthiaeddy, who, I believe, initially wrote this line of code, that is connected with a line of code in the <Profile/> component.

![Terminal Error Image](https://user-images.githubusercontent.com/32943879/169660869-25ed1fe7-9cc9-4120-a751-d400b7bb1f51.png)